src/common/memory/Memory.h

```diff
@@ -0,0 +1,115 @@
+/* Copyright (c) 2022 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License.
+ */
+#pragma once
+
+#include <jemalloc/jemalloc.h>
+
+#include <new>
+
+#include "common/base/Base.h"
+#include "common/memory/MemoryTracker.h"
+
+namespace nebula {
+namespace memory {
+
+inline ALWAYS_INLINE size_t alignToSizeT(std::align_val_t align) noexcept {
+  return static_cast<size_t>(align);
+}
+
+inline ALWAYS_INLINE void* newImpl(std::size_t size) {
+  void* ptr = malloc(size);
+
+  if (LIKELY(ptr != nullptr)) return ptr;
+
+  throw std::bad_alloc{};
```

Comment:
I will leave this for future improvement, It is a little tricky here, if the exception to throw new sometine,  it will recursive result in throw that exception.